haystack/nodes/summarizer/transformers.py

```diff
@@ -134,29 +135,22 @@ def predict(self, documents: List[Document], generate_single_summary: Optional[b
         These document can for example be retrieved via the Retriever.
 
         :param documents: Related documents (e.g. coming from a retriever) that the answer shall be conditioned on.
-        :param generate_single_summary: Whether to generate a single summary for all documents or one summary per document.
-                                        If set to "True", all docs will be joined to a single string that will then
-                                        be summarized.
-                                        Important: The summary will depend on the order of the supplied documents!
-        :return: List of Documents, where Document.text contains the summarization and Document.meta["context"]
-                 the original, not summarized text
+        :param generate_single_summary: This parameter is deprecated and will be removed in Haystack 1.12
```

Comment:
Let's add a mention to the fact that the DocumentMerger is taking over this task: 

```suggestion
        :param generate_single_summary: This parameter is deprecated and will be removed in Haystack 1.12. To obtain single summaries from multiple documents, consider using the [DocumentMerger](https://docs.haystack.deepset.ai/docs/document_merger)
```